### DISCUSSION TITLE
请问各位都是怎么判断歌曲是否有版权播放的？

### DISCUSSION THREAD

**Discussion by wood3n** [Binaryify/NeteaseCloudMusicApi#1775] [Q&A]
看了接口返回的`noCopyrightRcmd`并不能用准确判断歌曲是否能播放

**Comment by ice909** [comment]
这个接口
![image](https://github.com/Binaryify/NeteaseCloudMusicApi/assets/59855569/cb02c3fa-5ca9-4ba4-be68-2d12f00cd9c6)

**Reply by wood3n** [reply]
好的。想过用这个，但是在表格里面每条数据都得请求这个，感觉又不太合适

**Reply by ice909** [reply]
获取音乐url的时候，如果url为空应该也是无版权
